hack/utils.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -x
+
+function running_from_skipper() {
+   # The SKIPPER_UID environment variable is an indication that we are running on a skipper container.
+   [ -n "${SKIPPER_UID+x}" ]
+}
+
+function get_container_runtime_command() {
+    if running_from_skipper; then
+        # skipper creates the SKIPPER_DOCKER_GID env variable for a docker based build
+        CONTAINER_TOOL=$( [[ "$CONTAINER_RUNTIME_COMMAND" == ""docker"" ]] && echo "docker" || echo "podman-remote")
+    else 
+        CONTAINER_TOOL=$( command -v docker &> /dev/null && echo "docker" || echo "podman")
```

Comment:
In Makefile line 18 seems like `podman` is chosen over docker if both exist, but here seems like it is the opposite?

I don't think it makes a big change but may be worth staying consistent.

Reply:
please read my comment there